### DISCUSSION TITLE
Can't open file in VS Code but it works on CMD

### DISCUSSION THREAD

**Discussion by VictorValar** [Roche/pyreadstat#125] [Q&A] [answered]
Hello, how are you guys?

I'm new to Python so this may be a noobie question:
I'm trying to open a file in VS Code but it keeps saying the file does not exist, but if I try to run the code on CMD it opens it perfectly!

Any ideias of what is happening?

![image](https://user-images.githubusercontent.com/54588980/114253997-558cfa00-9983-11eb-92d3-7c87798bb531.png)

![image](https://user-images.githubusercontent.com/54588980/114254015-69386080-9983-11eb-85ce-94fb7da1314d.png)

**Comment by ofajardo** [comment, marked as answer]
In the command line you are doing cd into the working folder where the sav file is, thats why python finds it. 

In Vscode you are in another folder when executing, thats why it cannot find it.

When opening the sav file, give the full path (r'C:\blah\blah\fioe.sav) or, in the Vscode terminal cd into the working folder as you do in CMD
